benchmarks/benchmark_serving.py

```diff
@@ -392,39 +409,10 @@ async def benchmark(
         outputs=outputs,
         dur_s=benchmark_duration,
         tokenizer=tokenizer,
+        selected_percentile_metrics=selected_percentile_metrics,
+        selected_percentiles=selected_percentiles,
     )
 
-    print("{s:{c}^{n}}".format(s=' Serving Benchmark Result ', n=50, c='='))
```

Comment:
percentiles are printed in if-blocks below when the corresponding metric is specified by, e.g., --percentile-metrics=ttft,itl.